src/Discord.Net.Commands/Info/CommandInfo.cs

```diff
@@ -63,6 +63,9 @@ internal CommandInfo(CommandBuilder builder, ModuleInfo module, CommandService s
             _action = builder.Callback;
         }
 
+        public TPrecondition GetPrecondition<TPrecondition>() where TPrecondition : PreconditionAttribute =>
```

Comment:
Ah, I see. But if you *had* a base Precondition, then it would make sense to get all of the applied preconditions when you do `GetPreconditions<MyBasePrecondition>()` instead of none of them.